components/Modals/BuyingGuideModal.tsx

```diff
@@ -12,10 +12,6 @@ import { sliceAddress } from "@/lib/utils";
 import { buyWithUniswap } from "@/payments";
 import { USDC_POLYGON_CONTRACT_ADDRESS } from "@/utils";
```

Comment:
@har777 we also need USDC_MAINNET_CONTRACT_ADDRESS variability here

Reply:
good catch!

Reply:
also note that `USDC_POLYGON_CONTRACT_ADDRESS` works only on prod
I did this trick to default to MATIC on non-prod envs to anything is being displayed:
https://github.com/Glo-Foundation/glo-wallet/pull/305/files#diff-f54cb834b7fbc073729588ea3b3b774a25102a62640e8b5ffc8e03eeb6f1aa6eR39